packages/store/src/actions/symbols.ts

```diff
@@ -1,10 +1,18 @@
-export interface ActionDef<T = any, U = any> {
+export class StaticAction {
+  public static type: string;
+}
```

Comment:
The problem is that our actions are written this way.

```ts
export class Update {
  public static get type() {
    return 'ACTION';
  }
}
```

That is, we can not write an interface for our actions. Because it is not possible to write an interface for static fields, so that the script script does not swear that you are dispatching an action with a static field, you can specify a class as a type.